data-reconciliation-app/docs/adr/03-reconciliation-logic.md

```diff
@@ -11,37 +11,37 @@ Once data is ingested into the app, Members/Users will query for the data to be
 ## Requirements
 
 - Reconciliation is on each record.
-- Reconciled information will apply only to items a User has submitted data for.
-- Anytime a User requests a Reconciliation Report, Logic will be executed for this User on the current dataset stored in the network
+- Reconciled information will apply only to items for which a user has submitted data.
+- When a user requests for Reconciliation Report, logic on the current dataset stored in the network is executed for this user.
 - Result will not present other Users' information on a specific record, but just a comparison with them.
 
 ## When Reconciliation Happens
 
-This logic will be embedded on the reporting API, and anytime a member asks for their data to be reconciled (ask for a report), the application will do the necessary calculations based on the data this member has entered.
+This logic will be embedded in the reporting API, and anytime a member asks for their data to be reconciled (asks for a report), the application will do the necessary calculations based on the data this member has ingested.
```

Comment:
```suggestion
In `/report` endpoint, when a user requests for a Reconciliation Report, it will be generated by the data that user ingested previously.
```